src/concrete/ml/sklearn/neighbors.py

```diff
@@ -110,7 +110,7 @@ def load_dict(cls, metadata: Dict):
         obj._q_fit_X_quantizer = metadata["_q_fit_X_quantizer"]
         obj._q_fit_X = metadata["_q_fit_X"]
         obj._y = metadata["_y"]
-
+        obj._rounder = metadata["_rounder"]
```

Comment:
This is generating flaky tests: 
```
            elif not lsbs_to_remove.is_adjusted:
                message = (
                    "AutoRounders cannot be used before adjustment, "
                    "please call AutoRounder.adjust with the function that will be compiled "
                    "and provide the exact inputset that will be used for compilation"
                )
>               raise RuntimeError(message)
E               RuntimeError: AutoRounders cannot be used before adjustment, please call AutoRounder.adjust with the function that will be compiled and provide the exact inputset that will be used for compilation
```
AutoRounders must become a serialized  class from concrete side.

Reply:
have you taken a look at the class  ? could it be possible to just manually serialize all of its attributes, like we've done for some objects ?

also, we can't just re-create a new autorounder object when loading the model, am I wrong ? since they auto-rounder object is modified during fitting (right ?), re-creating the object would just be useless .. ? This is just to make sure we absolutely need to serialize it !